### PR TITLE
18 sass

### PR DESCRIPTION
**Pull request by BovkunLeonid**

### CODE REVIEW

File: Site_Laser/styles/styles.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@import "_main-txeme";
```
Comment (RomanIlkiv) [inline Site_Laser/styles/styles.scss:1]: no `_` in the import
it will work without it

File: Site_Laser/styles/_contact.scss [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    span {
+        text-decoration: underline;
+    }
+
+    p {
+        margin-top: 0;
+    }
+}
+
+h2 {
```
Comment (RomanIlkiv) [inline Site_Laser/styles/_contact.scss:55]: don't create such general styles
especially in the file related only to Contact US

File: Site_Laser/styles/_gallery.scss [hunk 3]
```diff
@@ -1,49 +1,53 @@
...
+    flex-direction: column;
+    flex-wrap: wrap;
+    justify-content: start;
+    height: 3900px;
+
+    img {
+    width: 50%;
+
+    &:hover {
+    @include speed;
```
Comment (RomanIlkiv) [inline Site_Laser/styles/_gallery.scss:16]: bad formatting
no space before @include

File: Site_Laser/styles/styles.scss (hunk 1 above)
Comment (BovkunLeonid) [reply Site_Laser/styles/styles.scss:1]: Ок

File: Site_Laser/styles/_contact.scss (hunk 2 above)
Comment (BovkunLeonid) [reply Site_Laser/styles/_contact.scss:55]: ок

File: Site_Laser/styles/_gallery.scss (hunk 3 above)
Comment (BovkunLeonid) [reply Site_Laser/styles/_gallery.scss:16]: ок

**Pull request merged**